### PR TITLE
Class02

### PR DESCRIPTION
**Pull request by racer97**
hw1 from racer97

### CODE REVIEW

**Comment by emma-oc** [comment]
Good job

**Comment by racer97** [comment]
HW2 updated

**Comment by racer97** [comment]
Please read the same pull request for ex5 and ex6

File: class02/exercise_5.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+# >>> 55
+# >>> 75
+# >>> 150
+
+# code up your solution here
+print('Question 4')
+list3 = [12, 15, 32, 42, 55, 75, 122, 132, 150, 180, 200]
+for num in list3:
+    if num > 150:
+        break;
```
Comment (emma-oc) [inline class02/exercise_5.py:82]: you can remove the `;` here as it's not needed

File: class02/exercise_6.py [hunk 2]
```diff
@@ -23,6 +23,9 @@ def calculation(a, b):
 	'''
 
 	# code up your solution here
+	summation = a + b
+	diff = a - b
+	return[summation, diff]
```
Comment (emma-oc) [inline class02/exercise_6.py:28]: this is returning a list, think how we can return two numbers

**Comment by emma-oc** [comment]
Good job! I added a few comment on really small things:)

File: class02/exercise_6.py (hunk 2 above)
Comment (racer97) [reply class02/exercise_6.py:28]: Fixed. Now returns a tuple.
Thanks for reviewing the homework.

**Pull request closed without merging**